src/index.js

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Navbar from './components/Navbar';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Books from './pages/Books';
+import Categories from './pages/Categories';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<App />} />
+        <Route path="/books" element={<Books />} />
+        <Route path="/categories" element={<Categories />} />
+      </Routes>
+    </BrowserRouter>
```

Comment:
- [ ] I have a couple of recommendations here. The first one is, usually, our App component wraps around most of the elements of our App, except context elements. So in this case, I would place the Routes and the Navbar inside of the App component.

- [ ] In that case, remove the Route `/books`, we're not using this route, instead, our `Books` component should render when we're on the homepage: `/`

Reply:
Recommendations fixed and project looks organized now. Thanks again @D4nielJ